tools/elk/bin/run-logstash

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+logstash "$@" &
```

Comment:
should we update the readme with starting logstash with this script?

Reply:
Maybe? :) It's not necessary and there's a chance this script may not work in some environments.